Library/Formula/thefuck.rb

```diff
@@ -45,10 +45,18 @@ class Thefuck < Formula
     sha256 "1a089279d5de2471c47624d4463f2e5b3fc6a2cf65045c39bf714fc461a25206"
   end
 
+  resource "pip" do
+    url "https://pypi.python.org/packages/source/p/pip/pip-8.0.3.tar.gz"
+    sha256 "30f98b66f3fe1069c529a491597d34a1c224a68640c82caf2ade5f88aa1405e8"
+  end
+
+  # Temporarily fix cache incompatibilities issues. FIXME: Remove this in 3.5!
+  patch :DATA
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!